src/base/private/profile_p.h

```diff
@@ -48,7 +48,12 @@ namespace Private
 
         virtual ~Profile() = default;
 
-        QString configurationName() const;
+		/**
+		 * @brief Configuration name suitable to use as directory name
+		 *
+		 * @return QCoreApplication::applicationName() with optional configuration name appended
+		 */
+		QString configurationName() const;
```

Comment:
Perhaps, `profileName()` (or simply `name()` since it is a member of `Profile` class) would be better?